prow/cmd/deck/main.go

```diff
@@ -41,10 +41,11 @@ import (
 )
 
 var (
-	configPath   = flag.String("config-path", "/etc/config/config", "Path to config.yaml.")
-	buildCluster = flag.String("build-cluster", "", "Path to file containing a YAML-marshalled kube.Cluster object. If empty, uses the local cluster.")
-	tideURL      = flag.String("tide-url", "", "Path to tide. If empty, do not serve tide data.")
-	hookURL      = flag.String("hook-url", "", "Path to hook plugin help endpoint.")
+	configPath      = flag.String("config-path", "/etc/config/config", "Path to config.yaml.")
+	buildCluster    = flag.String("build-cluster", "", "Path to file containing a YAML-marshalled kube.Cluster object. If empty, uses the local cluster.")
+	tideURL         = flag.String("tide-url", "", "Path to tide. If empty, do not serve tide data.")
+	hookURL         = flag.String("hook-url", "", "Path to hook plugin help endpoint.")
+	redirectToHTTPS = flag.Bool("redirect-to-https", false, "Enable x-forwarded-for http to https redirect.")
```

Comment:
`x-forwarded-for` != `x-forwarded-proto`

Reply:
oops, thanks.